### PR TITLE
Tesla Presence

### PR DESCRIPTION
**Pull request by N0chteil**
Proof:
Screenshot_307
Screenshot_308
Screenshot_309

### CODE REVIEW

**Comment by N0chteil** [comment]
Proof for store:
Screenshot_310
Screenshot_311

**Comment by Timeraa** [comment]
Elon, where's my Tesla?

File: websites/T/Tesla/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "id": "495901098926669825"
+  },
+  "contributors": [{
+    "name": "Brotツ",
+    "id": "495901098926669825"
+  }],
+  "service": "Tesla",
+  "description": {
+    "en": "Tesla, Inc. is an American electric vehicle and clean energy company based in Palo Alto, California. Tesla's current products include electric cars, battery energy storage from home to grid scale, solar panels and solar roof tiles, as well as other related products and services."
+  },
```
Comment (Bas950) [inline websites/T/Tesla/dist/metadata.json:14]:
```suggestion
  "description": {
    "en": "Tesla is accelerating the world's transition to sustainable energy with electric cars, solar and integrated renewable energy solutions for homes and businesses.",
    "nl": "Tesla versnelt de wereldwijde transitie naar duurzame energie, met elektrische auto's, zonnepanelen en geïntegreerde duurzame energieoplossingen voor woningen en bedrijven."
  },
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]
It's literally better than most of presences like this lol

**Review by Bas950 — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]
amazing presence

**Pull request merged**